### PR TITLE
ali: initial support to CrDroid 8

### PR DESCRIPTION
**Pull request by brunorolak**
all dts is on Cr sources

### CODE REVIEW

File: ali.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "buildtype": "Monthly",
+        "forum": "https://forum.xda-developers.com/t/rom-12-1-crdroid-8-6-for-moto-g6-ali-official-64-bit.4463359/",
+        "gapps": "https://nikgapps.com/",
+        "firmware": "",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "https://forum.xda-developers.com/t/twrp-unofficial-twrp-64-bit-3-6-1.4328145/",
+        "paypal": "",
+        "telegram": "https://t.me/Moto_G6",
+        "dt": "https://github.com/brunorolak/device_motorola_ali",
```
Comment (gwolf2u) [inline ali.json]: use crDroid organization links for dt, common and kernel

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: ali.json (hunk 1 above)
Comment (brunorolak) [reply ali.json]: close these pr I'll make another witch right repo, my draught is jumping on me and I missed to wrote properlly

Comment (gwolf2u) [reply ali.json]: you can force push
no need to close
I'll eventually squash if you push change commit

Comment (brunorolak) [reply ali.json]: oko fixed by forced push

File: ali.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "buildtype": "Monthly",
+        "forum": "https://forum.xda-developers.com/t/rom-12-1-crdroid-8-6-for-moto-g6-ali-official-64-bit.4463359/",
+        "gapps": "https://nikgapps.com/",
+        "firmware": "",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "https://forum.xda-developers.com/t/twrp-unofficial-twrp-64-bit-3-6-1.4328145/",
+        "paypal": "",
+        "telegram": "https://t.me/Moto_G6",
+        "dt": "https://github.com/crdroidandroid/android_device_motorola_ali",
```
Comment (gwolf2u) [inline ali.json:23]: encryption need to be forced
https://github.com/crdroidandroid/android_device_motorola_ali/blob/12.1/rootdir/etc/fstab.qcom#L8

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: ali.json (hunk 2 above)
Comment (brunorolak) [reply ali.json:23]: my device is SAR, encryptation works well

Comment (brunorolak) [reply ali.json:23]: done, I'll make and upload a newbuild with force encrypt

**Pull request merged**